src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuilds.java

```diff
@@ -94,8 +96,7 @@ public void onCompleted(AbstractBuild<?, ?> build, TaskListener listener) {
         duration);
 
     // Merge PR
-    StashBuildTrigger trig = StashBuildTrigger.getTrigger(build.getProject());
-    if (trig.getMergeOnSuccess() && build.getResult() == Result.SUCCESS) {
+    if (trigger.getMergeOnSuccess() && build.getResult() == Result.SUCCESS) {
```

Comment:
I didn't expect this based on the PR description :)
Are you 100% sure this is always the same?

Reply:
OK, just saw your comment above

Reply:
I have actually tested the merge functionality. The `trigger` variable comes from the constructor that is created by an instance of `StashBuildTrigger`. I believe it should be trusted.